src/main.js

```diff
@@ -361,12 +361,12 @@ export default class FeatureService {
     this._appendTokenIfExists(params)
 
     return new Promise((resolve) => {
-      this._requestJson(`${this._esriServiceOptions.url}/query?${params.toString()}`)
+      this._requestJson(`${this._esriServiceOptions.url}/query?${params.toString()}`, this._esriServiceOptions.fetchOptions)
```

Comment:
Rather than adding this as an argument every time is there a reason we can't add it in the `this._requestJson` function itself?

Reply:
Yes because in the project bounds fallback request, you don't want to send the headers as the request is going to a different domain than the one you're targeting that time.

Reply:
Actually, I've guarded for that already. I'll make the update and get back to you tomorrow.

Reply:
Sorry, upon reviewing the code again, the `_projectBounds` still does need to swap out the fetch options in the fallback case. We could add something like `_requestJson(url, noOptions)` and have that only be `true` in this case, but it makes more sense to me to mimic the `fetch` API of `fetch(url, init)` that is being called under the hood where `init` in our case is the semantically named `fetchOptions` so the `_requestJson(url, fetchOptions)`

What do you think @rowanwins?

Reply:
Having now looked at it a bit closer I think what you've done is reasonable, it's probably neither here nor there with the other `noOptions` proposal.